app/Schema/user_schema.py

```diff
@@ -36,10 +35,6 @@ def if_mobile_no_exist(mobile_number):
     return User.query.filter(User.mobile_number == mobile_number).first()
```

Comment:
usually return value boolean for the functions with name `if_*`.

Reply:
Yes, but we are using this functions in view.user for checking if the mobile number or email_id is duplicate(already taken) and returning the error response their.